dunwoody_disaster/FightSequence.py

```diff
@@ -12,6 +14,9 @@ def __init__(
         self.enemy = enemy
         self.widget = FightScreen(self)
 
+        self._winCallback = DD.unimplemented
+        self._loseCallback = DD.unimplemented
```

Comment:
@NoahBartell Would you be able to hook your defeat screen up to this API? You should be able to copy the victory screen architecture for the most part.

Reply:
@NoahBartell I am going to mark this as resolved so that we can get this functionality merged into main in hopes of avoiding some conflicts.